CONTRIBUTING.md

```diff
@@ -1,69 +1,106 @@
-# Contributing
+# Contributing to Identity Platform Bundle
 
 ## Overview
 
-This document explains the processes and practices recommended for contributing enhancements to
-this operator.
+This document illustrates the processes and practices recommended for
+contributing to the Identity Platform bundle.
 
-<!-- TEMPLATE-TODO: Update the URL for issue creation -->
+## Developing the bundle integration test
 
-- Generally, before developing enhancements to this charm, you should consider [opening an issue
```

Comment:
This section seems like a pretty standard part of all canonical CONTRIBUTING.md file, so unless I'm mistaken it should probably stay in.

Reply:
tbh, It is mainly for developing charms, and some of the canonical charms don't have this section. But let me add it back and adapt it a bit.